### PR TITLE
Add unit tests for aria-hidden to modal suite

### PR DESCRIPTION
**Pull request by kkirsche**
Add tests for `aria-hidden` to modal test suite

### CODE REVIEW

**Comment by twbs-savage** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: de34782e020d53c8bd0348b2eaa38b6829926273
Build details: https://travis-ci.org/twbs-savage/bootstrap/builds/52039397

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

File: js/tests/unit/modal.js [hunk 1]
```diff
@@ -86,6 +97,22 @@ $(function () {
       .bootstrapModal('show')
   })
 
+  QUnit.test('should set aria-hidden to true when hide is called', function (assert) {
+    var done = assert.async()
+
+    $('<div id="modal-test"/>')
+      .on('shown.bs.modal', function () {
+        assert.ok($('#modal-test').is(':visible'), 'modal visible')
+        assert.notEqual($('#modal-test').length, 0, 'modal inserted into dom')
```
Comment (cvrebert) [inline js/tests/unit/modal.js]: Why not strictEqual 1 instead?

Comment (kkirsche) [inline js/tests/unit/modal.js]: That does make more sense. Honestly that first part was just a copy from the 'should toggle when toggle is called' test

**Comment by kkirsche** [comment]
Rebased per @cvrebert's suggestion to use `strictEqual` for length of `1`

**Comment by twbs-savage** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: 30d8cdf6398f05a58afa303ac324201d08744b90
Build details: https://travis-ci.org/twbs-savage/bootstrap/builds/52047697

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Pull request merged**